src/views/registration/attendees/fields/checkbox.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * This template renders the Checkbox
+ *
+ * @version TBD
+ *
+ */
+$required      = isset( $field->required ) && 'on' === $field->required ? true : false;
+$field         = (array) $field;
+$attendee_id   = null;
+$value         = '';
+$is_restricted = false;
+$options       = null;
+
+if ( isset( $field['extra'] ) && ! empty( $field['extra']['options'] ) ) {
```

Comment:
lets use `Tribe__Utils__Array::get( array( 'extra', 'options' ) )` to avoid conditional